Source/UI/Activation/Scenes/EnterCode/EnterActivationCodeModels.swift

```diff
@@ -24,6 +24,10 @@ public enum EnterActivationCode {
             let sceneTitle: String
             let sceneDescription: String
             let confirmButton: String
+            
+            let notValidAlertTitle: String
+            let notValidAlertText: String
+            let notValidAlertButton: String
```

Comment:
Yes, I thought about it and eventually chose this option. The mapping can be done in the app via LocalizationProvider (as I have done it in our product while integrating these changes) to prevent duplication of the "OK", but I can imagine a language where "I understand" or "Check the code" can make more sense.